akka-cluster/src/main/scala/akka/cluster/ClusterReadView.scala

```diff
@@ -206,8 +206,7 @@ private[akka] class ClusterReadView(cluster: Cluster) extends Closeable {
   def close(): Unit = {
     _closed = true
     _cachedSelf = OptionVal.Some(self.copy(MemberStatus.Removed))
-    if (!eventBusListener.isTerminated)
-      eventBusListener ! PoisonPill
+    eventBusListener ! PoisonPill
```

Comment:
Wouldn't it be neater to do this check, even if `isTerminated` is no longer recommended?

Reply:
I assumed it was an optimisation only for concurrent calls to `close()`, didn't seem worth watching from the outside.

Reply:
we should remove the deprecation for `isTerminated`, it has been changed to internal API long time ago, and we can use

I'd like to remove the deprecation for 2.6

Reply:
Why do we need to wait for 2.6 to remove the deprecation?

Reply:
ok, we can remove it now
add the InternalApi annotation

Reply:
:+1: